### ISSUE TITLE
Import Error

### ISSUE THREAD

**Issue by ryan-xin** [brammitch/recharts-to-png#403]
Thanks for the library.

Any reason I'm getting this error when import?
`Cannot find module 'recharts-to-png' or its corresponding type declarations.`

Thanks :)

**Comment by arkadiuszbachorski** [comment]
I am also getting this error. Package directory contains no `dist`:

```
node_modules/recharts-to-png/
├── LICENSE
├── package.json
└── README.md
```

**Comment by brammitch** [comment]
@ryan-xin @arkadiuszbachorski looks like the build for 2.0.1 is broken. I'm out of town without access to my environment, so a fix will have to wait until this weekend. Can you verify that 2.0.0 is working?

**Comment by arkadiuszbachorski** [comment]
> @ryan-xin @arkadiuszbachorski looks like the build for 2.0.1 is broken. I'm out of town without access to my environment, so a fix will have to wait until this weekend. Can you verify that 2.0.0 is working?

Yes, 2.0.0 works correctly.

**Comment by brammitch** [comment]
Fixed in a6e234cbc8b46b71c2551a5de9c1b616dbd4da15
